Fix GCP e2e issues

## Duplicate key on:

 `2021/12/20 12:00:45 gcp_monitoring_alert_policies verify fields
    integration.go:140: 
        	Error Trace:	integration.go:140
        	            				integration_test.go:22
        	            				gcp_monitoring_alert_policies_test.go:13
        	Error:      	failed to create tables gcp_monitoring_alert_policies
        	Test:       	TestIntegrationMonitoringAlertPolicies
        	Messages:   	ERROR: duplicate key value violates unique constraint "pg_type_typname_nsp_index" (SQLSTATE 23505)
--- FAIL: TestIntegrationMonitoringAlertPolicies (0.03s)`




## Failed e2e check on:

`
    integration.go:148: verification failed for table gcp_storage_buckets; expected to have 1 but got 0 entries with one of the map[billing_requester_pays:false default_event_based_hold:false iam_configuration_bucket_policy_only_enabled:false iam_configuration_public_access_prevention:unspecified iam_configuration_uniform_bucket_level_access_enabled:false id:gcp-storage-buckets-cq-testing-integration kind:storage#bucket location_type:multi-region retention_policy_is_locked:false retention_policy_retention_period:0 satisfies_pzs:false 
`

--- a/resources/integration_tests/gcp_kms_keyrings_test.go
+++ b/resources/integration_tests/gcp_kms_keyrings_test.go
@@ -17,7 +17,7 @@ func TestIntegrationKmsKeyrings(t *testing.T) {
 		return providertest.ResourceIntegrationVerification{
 			Name: resources.KmsKeyrings().Name,
 			Filter: func(sq squirrel.SelectBuilder, res *providertest.ResourceIntegrationTestData) squirrel.SelectBuilder {
-				return sq.Where(squirrel.Like{"name": fmt.Sprintf("%%kms-keyring-%s%s-v2", res.Prefix, res.Suffix)})
+				return sq.Where(squirrel.Like{"name": fmt.Sprintf("%%kms-keyrg-%s%s-v2", res.Prefix, res.Suffix)})
 			},
 			ExpectedValues: []providertest.ExpectedValue{
 				{

--- a/resources/integration_tests/gcp_sql_instances_test.go
+++ b/resources/integration_tests/gcp_sql_instances_test.go
@@ -21,7 +21,7 @@ func TestIntegrationSQLInstances(t *testing.T) {
 			ExpectedValues: []providertest.ExpectedValue{{
 				Count: 1,
 				Data: map[string]interface{}{
-					"name":             fmt.Sprintf("sql-database-instance-%s%s-v2", res.Prefix, res.Suffix),
+					"name":             fmt.Sprintf("sql-database-inst-%s%s-v2", res.Prefix, res.Suffix),
 					"database_version": "POSTGRES_11",
 				},
 			}},

--- a/resources/integration_tests/gcp_storage_buckets_test.go
+++ b/resources/integration_tests/gcp_storage_buckets_test.go
@@ -29,7 +29,7 @@ func TestIntegrationStorageBuckets(t *testing.T) {
 						"billing_requester_pays":   false,
 						"default_event_based_hold": false,
 						"iam_configuration_bucket_policy_only_enabled":          false,
-						"iam_configuration_public_access_prevention":            "unspecified",
+						"iam_configuration_public_access_prevention":            "inherited",
 						"iam_configuration_uniform_bucket_level_access_enabled": false,
 						"retention_policy_is_locked":                            false,
 						"retention_policy_retention_period":                     float64(0),

--- a/resources/integration_tests/infra/gcp_kms_keyrings.tf
+++ b/resources/integration_tests/infra/gcp_kms_keyrings.tf
@@ -1,5 +1,5 @@
 resource "google_kms_key_ring" "gcp_kms_keyrings_keyring" {
-  name     = "kms-keyring-${var.test_prefix}${var.test_suffix}-v2"
+  name     = "kms-keyrg-${var.test_prefix}${var.test_suffix}-v2"
   location = "global"
 
 }

--- a/resources/integration_tests/infra/gcp_sql_instances.tf
+++ b/resources/integration_tests/infra/gcp_sql_instances.tf
@@ -1,5 +1,5 @@
 resource "google_sql_database_instance" "sql_db_instance" {
-  name                = "sql-database-instance-${var.test_prefix}${var.test_suffix}-v2"
+  name                = "sql-database-inst-${var.test_prefix}${var.test_suffix}-v2"
   database_version    = "POSTGRES_11"
   deletion_protection = false
 

